src/main/scala/edu/ie3/simona/config/SimonaConfig.scala

```diff
@@ -2902,9 +2978,9 @@ object SimonaConfig {
     }
 
     final case class Time(
-        endDateTime: java.lang.String,
-        schedulerReadyCheckWindow: scala.Option[scala.Int],
-        startDateTime: java.lang.String,
+        endDateTime: String = "2011-05-01T01:00:00Z",
+        schedulerReadyCheckWindow: Option[Int] = None,
+        startDateTime: String = "2011-05-01T00:00:00Z",
```

Comment:
Don't think that there should be default simulation start and end, as this is very individual to the specific simulation. But I realize this is how it was done beforehand, so maybe we should do this in a separate issue

Reply:
see #1179

Reply:
Thanks!